<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CXLazerDirectionDialog</class>
 <widget class="QDialog" name="CXLazerDirectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Переехать</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="CXTouchButton" name="mAbsoluteButton">
     <property name="text">
      <string>Абсолютное</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CXTouchButton" name="mRelativeButton">
     <property name="text">
      <string>Относительное</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CXTouchButton" name="mCancelButton">
     <property name="text">
      <string>Отменить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CXTouchButton</class>
   <extends>QPushButton</extends>
   <header>cxtouchbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
